src/java.base/share/classes/sun/security/ec/ECKeyFactory.java

```diff
@@ -219,6 +219,8 @@ private PublicKey implGeneratePublic(KeySpec keySpec)
                 }
                 yield new ECPublicKeyImpl(p8key.getPubKeyEncoded());
             }
+            case null -> throw new InvalidKeySpecException(
+                "keySpec most not be null");
```

Comment:
`s/most/must/`. Add every other `case null` in this PR.

Reply:
Sigh.. one typo that is cut and pasted repeatedly.  Sure.